jobs/rocoto/vrfy.sh

```diff
@@ -69,7 +69,7 @@ if [ ${VRFYFITS} = "YES" -a ${CDUMP} = ${CDFNL} -a ${CDATE} != ${SDATE} ]; then
     xdate=$(${NDATE} -${VBACKUP_FITS} ${CDATE})
 
     export RUN_ENVIR_SAVE=${RUN_ENVIR}
-    export RUN_ENVIR=${OUTPUT_FILE}
+    export RUN_ENVIR="netcdf"
```

Comment:
Is this used in some downstream script we don't control yet? Or is this another control we can remove?

Reply:
It is used in downstream script. Can not be removed.